devicetypes/smartthings/fibaro-dimmer.src/fibaro-dimmer.groovy

```diff
@@ -83,7 +94,7 @@ def parse(String description) {
 		value:  description
 	]
 	def result
-	def cmd = zwave.parse(description, [0x26: 1, 0x70: 2, 072: 2])
```

Comment:
Interesting, this had a bug `072`

Reply:
Although this DTH doesn't have a handler for `manufacturerspecificv2`. Perhaps that CC should be removed from the list that are parsed?

Reply:
Aah I thought it did. Will remove